infrastructure/src/main/java/org/corfudb/infrastructure/SequencerServer.java

```diff
@@ -248,7 +250,8 @@ private String getConflictHashCode(UUID streamId, byte[] conflictParam) {
      *     cause.
      */
     private TokenType txnCanCommit(TxResolutionInfo txInfo, /** Input. */
-                                  AtomicReference<byte[]> conflictKey /** Output. */) {
+                                  AtomicReference<byte[]> conflictKey, /** Output. */
+                                  AtomicReference<UUID> conflictStreamID /** Output. */) {
```

Comment:
It seems like we just return a conflict and not all conflicts. Do we care about all conflicts?